core/commands/IssueUpdateCommand.php

```diff
@@ -0,0 +1,489 @@
+
+        # Update custom field values.
+        mci_issue_set_custom_fields( $this->updated_issue->id, $this->custom_fields_to_set );
+
+        # Add a bug note if there is one.
+        if( $this->bug_note->note || helper_duration_to_minutes( $this->bug_note->time_tracking ) > 0 ) {
+            $t_bugnote_id = bugnote_add( $this->updated_issue->id, $this->bug_note->note, $this->bug_note->time_tracking, $this->bug_note->view_state == VS_PRIVATE, 0, '', null, false );
+            bugnote_process_mentions( $this->updated_issue->id, $t_bugnote_id, $this->bug_note->note );
+        }
+
+        # Add a duplicate relationship if requested.
+        if( $this->updated_issue->duplicate_id != 0 ) {
+            relationship_upsert( $this->updated_issue->id, $this->updated_issue->duplicate_id, BUG_DUPLICATE, /* email_for_source */ false );
+
+            if( user_exists( $this->existing_issue->reporter_id ) ) {
+                bug_monitor( $this->updated_issue->duplicate_id, $this->existing_issue->reporter_id );
+            }
+            if( user_exists( $this->existing_issue->handler_id ) ) {
+                bug_monitor( $this->updated_issue->duplicate_id, $this->existing_issue->handler_id );
+            }
+
+            bug_monitor_copy( $this->updated_issue->id, $this->updated_issue->duplicate_id );
+        }
+
+        event_signal( 'EVENT_UPDATE_BUG', array( $this->existing_issue, $this->updated_issue ) );
+
+        # Allow a custom function to respond to the modifications made to the bug. Note
+        # that custom functions are being deprecated in MantisBT. You should migrate to
+        # the new plugin system instead.
+        helper_call_custom_function( 'issue_update_notify', array( $this->updated_issue->id ) );
+
+        # Send a notification of changes via email.
+        if( $this->resolve_issue ) {
+            email_resolved( $this->updated_issue->id );
+            email_relationship_child_resolved( $this->updated_issue->id );
+        } else if( $this->close_issue ) {
+            email_close( $this->updated_issue->id );
+            email_relationship_child_closed( $this->updated_issue->id );
+        } else if( $this->reopen_issue ) {
+            email_bug_reopened( $this->updated_issue->id );
+        } else if( $this->existing_issue->handler_id != $this->updated_issue->handler_id ) {
+            email_owner_changed( $this->updated_issue->id, $this->existing_issue->handler_id, $this->updated_issue->handler_id );
+        } else if( $this->existing_issue->status != $this->updated_issue->status ) {
+            $t_new_status_label = MantisEnum::getLabel( config_get( 'status_enum_string' ), $this->updated_issue->status );
+            $t_new_status_label = str_replace( ' ', '_', $t_new_status_label );
+            email_bug_status_changed( $this->updated_issue->id, $t_new_status_label );
+        } else {
+            email_bug_updated( $this->updated_issue->id );
+        }
+
+        return array( TRUE );
```

Comment:
The command's returns should depend on an option, e.g. `return_issue`.

- Web UI will set it to false to get an empty array.
- REST API would set it to true to get the update issue details.

See `rest_issue_update` method for the expected input and output format for the REST API.  This command should be updated to mirror what REST API needs and be used from there.

Reply:
Implemented.